clash-prelude/src/Clash/Annotations/TH.hs

```diff
@@ -1,14 +1,14 @@
 {-|
 
 This module can automatically generate TopEntity definitions from 'Clash.NamedTypes'
-annotations. Annotations involving data/type families must be inspected for correctness.
+annotations. Annotations involving data\/type families must be inspected for correctness.
 Not all cases can be handled with automatic generation due to the difficulty of type manipulation
-in template Haskell. In particular annotations _inside_ the following is unlikely to work:
+in template Haskell. In particular annotations __inside__ the following is unlikely to work:
 
-- Data/type family referencing other data/type families.
+- Data\/type family referencing other data\/type families.
 - Annotations inside recursive data types
 - Clock constraints other than a single HiddenClockResetEnable. (You can still
-  use arbitrary explicit clock/reset/enables!)
+  use arbitrary explicit clock\/reset\/enables!)
 
 See "Clash.Tests.TopEntityGeneration" for more examples.
```

Comment:
It would appear to refer to

https://github.com/clash-lang/clash-compiler/blob/master/clash-prelude/tests/Clash/Tests/TopEntityGeneration.hs

but that's not part of any distribution, right? It's not in `src/`, it's in `tests/`. Do we really want to link to a Github repository?